src/components/App.tsx

```diff
@@ -44,6 +45,7 @@ const App = ({
       <Route exact path={`/${localeParam}/${winterParam}/selected`} component={SelectedAreasPage} />
       <Route exact path={`/${localeParam}/${winterParam}/form`} component={WinterFormPage} />
       <Route exact path={`/${localeParam}/${winterParam}/form/:tab`} component={WinterFormPage} />
+      <Route component={NotFoundPage} />
       <Redirect to="/" />
```

Comment:
This `Redirect` is no longer needed, isn't it?

Reply:
Correct, removed